### PR TITLE
Remove unnecessary else

### PR DESCRIPTION
**Pull request by hezhizhen**

### CODE REVIEW

**Comment by gandarez** [comment]
@hezhizhen thanks for opening this PR. Please read our `CONTRIBUTING` document [here](https://github.com/wakatime/wakatime-cli/blob/develop/CONTRIBUTING.md#branching-stratgegy) and [here](https://github.com/wakatime/wakatime-cli/blob/develop/CONTRIBUTING.md#pull-requests) to get this PR approved. It also has failing tests because linting problems.

File: pkg/ini/ini.go [hunk 1]
```diff
@@ -183,11 +183,10 @@ func WakaHomeDir() (string, error) {
 	home, exists := os.LookupEnv("WAKATIME_HOME")
 	if exists && home != "" {
 		home, err := homedir.Expand(home)
-		if err != nil {
```
Comment (alanhamlett) [inline pkg/ini/ini.go:186]: I think the convention is to always check `if err != nil` and never `if err == nil`?

**Comment by hezhizhen** [comment]
> @hezhizhen thanks for opening this PR. Please read our `CONTRIBUTING` document [here](https://github.com/wakatime/wakatime-cli/blob/develop/CONTRIBUTING.md#branching-stratgegy) and [here](https://github.com/wakatime/wakatime-cli/blob/develop/CONTRIBUTING.md#pull-requests) to get this PR approved. It also has failing tests because linting problems.

Sorry for replying late. I'll fix it later.

File: pkg/ini/ini.go (hunk 1 above)
Comment (hezhizhen) [reply pkg/ini/ini.go:186]: Not always. The key is to return short cases first.

**Comment by alanhamlett** [comment]
Implemented with #755.

**Pull request closed without merging**